apps/console/src/features/secrets/components/secrets-list.tsx

```diff
@@ -318,16 +448,57 @@ const SecretsList: FC<SecretsListProps> = (props: SecretsListProps): ReactElemen
             }
             { secretList?.length > 0
                 ? (
-                    <Fragment>
+                    <ListLayout
+                        advancedSearch={ (
+                            <AdvancedSearchWithBasicFilters
+                                onFilter={ handleSecretsFilter }
+                                filterAttributeOptions={ [
+                                    {
+                                        key: 0,
+                                        text: t("common:name"),
+                                        value: "name"
+                                    }
+                                ] }
+                                filterAttributePlaceholder={
+                                    t("console:develop.features.secrets.advancedSearch.form" +
+                                        ".inputs.filterAttribute.placeholder")
+                                }
+                                filterConditionsPlaceholder={
+                                    t("console:develop.features.secrets.advancedSearch.form" +
+                                        ".inputs.filterCondition.placeholder")
+                                }
+                                filterValuePlaceholder={
+                                    t("console:develop.features.secrets.advancedSearch.form.inputs.filterValue" +
+                                    ".placeholder")
+                                }
+                                placeholder={ t("console:develop.features.secrets.advancedSearch.placeholder") }
+                                defaultSearchAttribute="name"
+                                defaultSearchOperator="co"
+                                data-testid={ `${ testId }-list-advanced-search` }
+                            />
+                        ) }
+                        currentListSize={ secretList.length }
+                        onPageChange={ () => 
+                            void 0
+                        }
```

Comment:
Let's format this properly.

```suggestion
                        onPageChange={ () => void 0 }
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/2946/commits/fd8b8b5fa0fceede98d85ab312f115342b9535bf